apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -141,7 +141,7 @@ defmodule Aecore.Chain.Worker do
     end
   end
 
-  @spec get_header(binary()) :: Block.t() | {:error, reason()}
+  @spec get_header(binary()) :: Header.t() | {:error, reason()}
```

Comment:
can you rename this to get_header_by_hash maybe, to describe what it actually does?